appservice/src/createAppService/AppServicePlanSkuStep.ts

```diff
@@ -59,46 +60,43 @@ export class AppServicePlanSkuStep extends AzureWizardPromptStep<IAppServiceWiza
     }
 
     private getRecommendedSkus(): ExtendedSkuDescription[] {
+        const recommendedGroup: string = localize('recommendedGroup', 'Recommended');
         return [
             {
                 name: 'F1',
                 tier: 'Free',
                 size: 'F1',
                 family: 'F',
                 capacity: 1,
-                label: localize('freeLabel', 'Free'),
-                description: localize('freeDescription', 'Try out Azure at no cost')
+                label: localize('freeLabel', 'Free (F1)'),
+                description: localize('freeDescription', 'Try out Azure at no cost'),
+                group: recommendedGroup
             },
             {
                 name: 'B1',
                 tier: 'Basic',
                 size: 'B1',
                 family: 'B',
                 capacity: 1,
-                label: localize('basicLabel', 'Basic'),
-                description: localize('basicDescription', 'Develop and test')
+                label: localize('basicLabel', 'Basic (B1)'),
+                description: localize('basicDescription', 'Develop and test'),
+                group: recommendedGroup
             },
             {
                 name: 'P1v2',
                 tier: 'Premium V2',
                 size: 'P1v2',
                 family: 'Pv2',
                 capacity: 1,
-                label: localize('premiumLabel', 'Premium'),
-                description: localize('premiumDescription', 'Use in production')
+                label: localize('premiumLabel', 'Premium (P1V2)'),
```

Comment:
```suggestion
                label: localize('premiumLabel', 'Premium (P1v2)'),
```

So that we match the other picks below like P2v2